ape_aws/kms/_cli.py

```diff
@@ -54,7 +63,103 @@ def create_key(
     cli_ctx.logger.success(f"Key created successfully with ID: {key_id}")
 
 
-# TODO: Add `ape aws kms import`
+@kms.command(name="import")
+@ape_cli_context()
+@click.option(
+    "-p",
+    "--private-key",
+    "private_key_path",
+    type=click.Path(),
+    help="The private key you intend to import",
+    metavar="Path",
```

Comment:
I don't think you need to set a metavar everywhere, click does it automatically

Reply:
removed all of these